### ISSUE TITLE
batch_size introduced before discussing batch normalization

### ISSUE THREAD

**Issue by cpranav93** [carpentries-incubator/deep-learning-intro#279]
In episode 3 we introduce batch_size as a parameter of the model before we introduce batch normalization. We can run the model without this parameter (as batch_size is anyways auto-used) and introduce it later in the lesson - we can mention when we introduce batch_normalization that batch_size was automatically being used.

**Comment by svenvanderburg** [comment]
This is introduced with #106, therefore I am closing this issue.
